app/src/main/java/com/github/multimatum_team/multimatum/MainActivity.kt

```diff
@@ -145,6 +143,24 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    /*
+    Overriding this function allows the app to start the QR-Code scanner immediately
+    if the camera permission is granted.
+     */
+    override fun onRequestPermissionsResult(
+        requestCode: Int,
+        permissions: Array<out String>,
+        grantResults: IntArray
+    ) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
+        if(requestCode == 123) {
```

Comment:
What is that code (123)? It should probably be a constant, especially if it appears elsewhere.

Reply:
Good point